aboutstyle.css

```
@@ -17,6 +17,12 @@ nav{
     align-items: center;
     flex-direction: row;
 }
+.navLists :hover{
+    background-color: rgb(236, 225, 17);
```

```suggestion
    background-color: pink;
```
what do you think about changing hover color to pink?

---

no no i am fine with current